docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "2.1"
+
+services:
+  sonarqube:
+    depends_on:
+      - db
+    image: sonarqube-with-community-branch-plugin:7.9-community
```

Comment:
This would need to be renamed to the official docker image name, if the repo is added to dockerhub